### PR TITLE
CB Unmanned Stamp H743

### PR DESCRIPTION
**Pull request by andyp1per**
Needs https://github.com/ArduPilot/ChibiOS/pull/88

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/StampH743/hwdef.dat [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+# enable DFU reboot for installing bootloader
+# note that if firmware is build with --secure-bl then DFU is
+# disabled
+ENABLE_DFU_BOOT 1
+
+# build ABIN for flash-from-bootloader support:
+env BUILD_ABIN True
+
+# ethernet consumes a big chunk of flash
+include ../include/minimize_common.inc
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/StampH743/hwdef.dat:236]: likely better to change env OPTIMIZE to -Os

**Review by tridge — CHANGES_REQUESTED** [review]
needs module double shuffle

**Comment by tridge** [comment]
How is testing going?

**Comment by andyp1per** [comment]
@tridge it needs a new rev - the LDO GPIOs don't have power

File: libraries/AP_HAL_ChibiOS/hwdef/StampH743/hwdef.dat (hunk 1 above)
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/StampH743/hwdef.dat]: Fixed

**Comment by andyp1per** [comment]
Testing nearly complete

**Review by Hwurzburg — APPROVED** [review]
lgtm

**Review by tridge — APPROVED** [review]

**Pull request merged**